Setup-OpenResty.md

```diff
@@ -11,6 +11,20 @@ tar zxpf luarocks-3.3.1.tar.gz
 cd luarocks-3.3.1
 ```
 
+
+
+build and install LuaRocks:
+```
+./configure --prefix=/usr/local/openresty/luarocks --with-lua=/usr/local/openresty/luajit/ --with-lua-include=/usr/local/openresty/luajit/include/luajit-2.1 --with-lua-lib=/usr/local/openresty/lualib
+
+make -j8 && sudo make install
+```
+
+> Cannot find the unzip command, just execute the apt installation
+```
+sudo apt install unzip zip
+```
+
 To configure LuaRocks for OpenResty, run this command:
```

Comment:
Actually, we already have the build and installation documentation of LuaRocks [here](https://github.com/casbin/lua-casbin/blob/master/Setup-OpenResty.md#:~:text=To%20configure%20LuaRocks,sudo%20make%20install) under the configure LuaRocks section. Is there any reason to change this?

Reply:
@rushitote actually we already have: https://github.com/casbin-lua/lua-resty-casbin , can we remove this `Setup-OpenResty.md` now? (we can merge its diff to: https://github.com/casbin-lua/lua-resty-casbin if there are any)

Reply:
@hsluoyz Yeah, I think we can do that and provide a link to lua-resty-casbin in the README.

Reply:
![17cf0924-88c7-4691-8a29-4e11c2073639](https://user-images.githubusercontent.com/14959876/129904914-9ef9c610-3328-4a1a-8a47-22c7ce8d9b5a.png)

Reply:
@Tinywan Thanks, I will update the setup doc with the unzip dependency.

Reply:
@rushitote thanksZ!